pkgs/top-level/all-packages.nix

```diff
@@ -881,7 +881,7 @@ in
 
   mcrypt = callPackage ../tools/misc/mcrypt { };
 
-  mongodb-tools = goPackages.mongo-tools.bin // { outputs = [ "bin" ]; };
+  mongodb-tools = (callPackage ../tools/misc/mongodb-tools { }).bin;
```

Comment:
Is it ok like this or maybe we want to produce every otutput? I couldn't figure it out and make 2 styles of it to make it consistent later.